.github/workflows/README.md

```diff
@@ -24,11 +23,10 @@ Run from `rust.yml` unless stated otherwise. Total 21 jobs but
 10. `Bench`
 11. `ASAN`
 12. `WASM`
-13. `schemars`
-14. `Arch32bit`
-15. `Cross`
-16. `Embedded`
-17. `Kani`
-18. `Coveralls` - run by `coveralls.yml`
-19. `release` - run by `release.yml`
-20. `labeler` - run by `manage-pr.yml`
+13. `Arch32bit`
+14. `Cross`
+15. `Embedded`
+16. `Kani`
+17. `Coveralls` - run by `coveralls.yml`
+18. `release` - run by `release.yml`
+19. `labeler` - run by `manage-pr.yml`
```

Comment:
These horrible diffs can be solved by using all lists items as `1.`:

```md
1. foo
1. bar
1. anotherfoo
```

which renders as

1. foo
1. bar
1. anotherfoo

Reply:
Yeah I considered that, but being that the main point is show the total I thought it was worth the effort to use numbers. I personally don't look at the rendered version of the file.